### DISCUSSION TITLE
SONOS: import favorite radio stations etc.

### DISCUSSION THREAD

**Discussion by Symbiot78** [music-assistant/support#953] [Feature requests and ideas]
Or allow us to add urls to radio stations section.

or some other way to play a url stream.

**Comment by OzGav** [comment]
You can create your own playlists and put the urls in there

**Comment by Symbiot78** [comment]
@OzGav  -  sorry for being a muppet... but how would you do that in MA ?

**Reply by OzGav** [reply]
Have a look here https://github.com/music-assistant/hass-music-assistant/discussions/661

In this discussions area there are HOW TO and TROUBLESHOOTING threads to help you

**Reply by OzGav** [reply]
Actually just saw your other post so you already know how to use m3u files

**Reply by OzGav** [reply]
Just in case I didn't understand your question. If you select any track in MA you can select the menu on the far right and then ADD TO PLAYLIST

**Reply by Symbiot78** [reply]
thanks.  that will not work for radiostations.. at least not coming from a m3u file.
I can click the 3 dots next to a radiostation and there's a 'add to library'. Clicking that doesn't seem to do anything though.

**Reply by OzGav** [reply]
Your two discussions seem related. If you have a URL stream for a radio station, as you have discovered, you can add that to a playlist. The RADIO tab is for supported RADIO station music providers (currently only TuneIn).

So in terms of this feature request I think your option 3 “ or some other way to play a url stream.” is already implemented?

**Reply by OzGav** [reply]
> I can click the 3 dots next to a radiostation and there's a 'add to library'. Clicking that doesn't seem to do anything though.

It adds it your favourites so if you click the heart at the top only those “added to library” will show

**Comment by OzGav** [comment]
Relates to v1
